plugins/logitech-bulkcontroller/device_common.h

```diff
@@ -0,0 +1,61 @@
+//
+//  device_common.h
+//  ProtocDemo
+//
+//  Created by Edwin Estrada on 7/29/21.
+//  Copyright © 2021 Edwin Estrada. All rights reserved.
+//
```

Comment:
no C++ comments please, and this needs the SPDX licence header

Reply:
Fixed